DuckDuckGo/ConfigurationDebugViewController.swift

```diff
@@ -169,7 +169,6 @@ class ConfigurationDebugViewController: UITableViewController {
         }
     }
 
-    // swiftlint:disable:next cyclomatic_complexity
```

Comment:
This was triggering a swiftlint alert.  I didn't introduce it but it was failing in the CI tests so I have to address this here.